exporter/datadogexporter/internal/logs/translator.go

```diff
@@ -91,20 +100,20 @@ func Transform(lr plog.LogRecord, res pcommon.Resource, sendLogRecordBody bool)
 		status = lr.SeverityText()
 		l.AdditionalProperties[otelSeverityText] = lr.SeverityText()
 	} else if lr.SeverityNumber() != 0 {
-		status = derviveStatusFromSeverityNumber(lr.SeverityNumber())
+		status = statusFromSeverityNumber(lr.SeverityNumber())
 	}
 	l.AdditionalProperties[ddStatus] = status
 	if lr.SeverityNumber() != 0 {
 		l.AdditionalProperties[otelSeverityNumber] = strconv.Itoa(int(lr.SeverityNumber()))
 	}
-	// for datadog to use the same timestamp we need to set the additional property of "@timestamp"
+	// for Datadog to use the same timestamp we need to set the additional property of "@timestamp"
 	if lr.Timestamp() != 0 {
 		// we are retaining the nano second precision in this property
 		l.AdditionalProperties[otelTimestamp] = strconv.FormatInt(lr.Timestamp().AsTime().UnixNano(), 10)
 		l.AdditionalProperties[ddTimestamp] = lr.Timestamp().AsTime().Format(time.RFC3339)
 	}
-	if sendLogRecordBody {
-		// set the Message
+	if l.Message == "" {
+		// set the Message from body as it isn't present as AdditionalProperties
```

Comment:
```suggestion
		// set the Message to the Body in case it wasn't already parsed as part of the attributes
```
The reason I am asking for this change is because we have been looking for the body to be parsed as part of the log's attributes. If it wasn't, we fall back to the body field.